internal/handler/msg_aggregate.go

```diff
@@ -281,6 +282,18 @@ func (h *Handler) MsgAggregate(ctx context.Context, msg *wire.OpMsg) (*wire.OpMs
 			qp.Filter = filter
 		}
 
+		if !h.EnableExperimentalPushdown && filter != nil {
```

Comment:
If `h.DisablePushdown == true`, is  `h.EnableExperimentalPushdown == true` intended to override it by enabling pushdown?

Reply:
I would say we should check both are not true at the same time in the main.go